apollo-portal/src/main/resources/static/config/history.html

```diff
@@ -104,13 +104,22 @@ <h6 class="col-md-5 text-right" ng-bind="releaseHistory.releaseTimeFormatted"></
                     <div class="panel-heading">
 
                         <span ng-bind="history.releaseTitle"></span>
+                        <span ng-if="history.isReleaseAbandoned">{{'Config.History.Abandoned' | translate }}</span>
```

Comment:
Yes, I add a class="label label-warning", now
![image](https://user-images.githubusercontent.com/22030674/83970984-48d6ea00-a90b-11ea-9558-6874a4ddc28b.png)

Reply:
Yes, I add a class="label label-warning no-radius", now it shows like this
![image](https://user-images.githubusercontent.com/22030674/83971702-6443f400-a90f-11ea-8463-ef066a764b70.png)
or it can get better?

Reply:
this looks ok